updates/2016-04-05.yml

```diff
@@ -0,0 +1,14 @@
+changed:
+  -
+    title: "Emails in Auth0"
+    tags:
+      - email
+    description: |
+      Updated the page to reflect new rules on using Auth0's built in email provider. Moved content on customizing emails to a new page.
+added:
+  -
+    title: "Customizing Your Emails"
```

Comment:
We dont need to be this detailed with the updates. Just make a single update that describes all the changes to the the email content.